frameworks/g-cloud-9/questions/services/template.yml

```diff
@@ -0,0 +1,55 @@
+name: "question_name"
```

Comment:
This template feels like a useful thing for developers, but we probably don't need it committed to the repo.

Or if we do want it it should be somewhere above the `frameworks` folder in the hierarchy, easily findable for all future frameworks and not interfering with any particular framework.